deployment/deployment.json

```diff
@@ -3020,6 +3020,28 @@
             "query-id": "ANz3TpZdY2syZGQvGA85ANNG7KiSWdPmv55kP4H4sRPJ"
           }
         }
+      },
+      "evm-zksync": {
+        "network": "ethereum",
```

Comment:
this should be `zksync`?

Reply:
Yeah, I need to make the change to the subgraph-cli tool. It fails if the network is not recognized